### ISSUE TITLE
128x128 and 64x64 icons is not installed by 'make install'

### ISSUE THREAD

**Issue by sl1pkn07** [fontforge/fontforge#3053]
`make install` not install 128x128 and 64x64 icons

```
---snip---
make[3]: No se hace nada para 'install-exec-am'.
 /usr/bin/mkdir -p '/tmp/makepkg/fontforge/pkg/fontforge/usr/share/applications'
 /usr/bin/mkdir -p '/tmp/makepkg/fontforge/pkg/fontforge/usr/share/icons/hicolor/22x22/apps'
 /usr/bin/mkdir -p '/tmp/makepkg/fontforge/pkg/fontforge/usr/share/icons/hicolor/24x24/apps'
 /usr/bin/mkdir -p '/tmp/makepkg/fontforge/pkg/fontforge/usr/share/icons/hicolor/16x16/apps'
 /usr/bin/mkdir -p '/tmp/makepkg/fontforge/pkg/fontforge/usr/share/icons/hicolor/32x32/apps'
 /usr/bin/mkdir -p '/tmp/makepkg/fontforge/pkg/fontforge/usr/share/icons/hicolor/256x256/apps'
 /usr/bin/mkdir -p '/tmp/makepkg/fontforge/pkg/fontforge/usr/share/icons/hicolor/scalable/apps'
 /usr/bin/mkdir -p '/tmp/makepkg/fontforge/pkg/fontforge/usr/share/icons/hicolor/48x48/apps'
 /usr/bin/mkdir -p '/tmp/makepkg/fontforge/pkg/fontforge/usr/share/mime/packages'
 /usr/bin/mkdir -p '/tmp/makepkg/fontforge/pkg/fontforge/usr/share/icons/hicolor/scalable/apps'
 /usr/bin/install -c -m 644 fontforge.desktop '/tmp/makepkg/fontforge/pkg/fontforge/usr/share/applications'
 /usr/bin/install -c -m 644 icons/22x22/apps/fontforge.png '/tmp/makepkg/fontforge/pkg/fontforge/usr/share/icons/hicolor/22x22/apps'
 /usr/bin/install -c -m 644 icons/32x32/apps/fontforge.png '/tmp/makepkg/fontforge/pkg/fontforge/usr/share/icons/hicolor/32x32/apps'
 /usr/bin/install -c -m 644 icons/24x24/apps/fontforge.png '/tmp/makepkg/fontforge/pkg/fontforge/usr/share/icons/hicolor/24x24/apps'
 /usr/bin/install -c -m 644 icons/48x48/apps/fontforge.png '/tmp/makepkg/fontforge/pkg/fontforge/usr/share/icons/hicolor/48x48/apps'
 /usr/bin/install -c -m 644 icons/256x256/apps/fontforge.png '/tmp/makepkg/fontforge/pkg/fontforge/usr/share/icons/hicolor/256x256/apps'
 /usr/bin/install -c -m 644 fontforge.xml '/tmp/makepkg/fontforge/pkg/fontforge/usr/share/mime/packages'
 /usr/bin/install -c -m 644 icons/scalable/apps/fontforge.svg '/tmp/makepkg/fontforge/pkg/fontforge/usr/share/icons/hicolor/scalable/apps'
 /usr/bin/install -c -m 644 icons/16x16/apps/fontforge.png '/tmp/makepkg/fontforge/pkg/fontforge/usr/share/icons/hicolor/16x16/apps'
make  install-data-hook
---snip---
```

https://github.com/fontforge/fontforge/tree/master/desktop/icons

sources downloaded from
https://github.com/fontforge/fontforge/archive/20161012.tar.gz

Greetings

**Comment by frank-trampe** [comment]
I'll fix it this week if nobody else gets to it first.

**Comment by jtanx** [comment]
Already fixed by #3103
